### ISSUE TITLE
polohu odberneho miesta zistovat podla GPS

### ISSUE THREAD

**Issue by 4ndrej** [psekan/somvrade#10]
polohu odberneho miesta by to mohlo zistovat podla GPS, zaokruhlit na desiatky metrov a zobrazit na mape - budu vznikat pekne zhluky okolo odbernych miest a netreba vypisovat miesto

tiez sa eliminuju zjavne blbosti

**Comment by mesiarm** [comment]
Súhlasím, pri zadávaní odberného miesta by mohli byť polia pre long, lat súradnice + jednoduchý návod ako ich skopírovať z Google Maps, neskôr ak bude čas sa to dá zlepšiť na výber na mape.

**Comment by 4ndrej** [comment]
myslel som ze by to mohla zistovat rovno stranka cez [geolokacne api](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API), pri rucnom opisovani tam vznikne tona chyb

**Comment by mirelon** [comment]
Pozrite #13 pre geo data odbernych miest v KE a kus kodu pre pridavanie farebnych markerov podla casu na mapu

**Comment by thebrownfox** [comment]
@4ndrej Geolocation vrati polohu pouzivatela, ktora je urcite ina ako poloha administratora, ktory ju tam zadava + potrebujes to riesit mozno aj davkovo, najlepsie cez geojson alebo nejaky format.

**Comment by 4ndrej** [comment]
- myslel som to tak ze pridem pred odberne miesto, kliknem ze som tu, zaznamena to moju polohu a pripadne kliknem ze je tu predomnou +/- 30 ludi
- alternativne este pri odchode kliknem ze odchadzam
- krabicka by si mohla spocitat kedy som daval info ze som prisiel (lokalna cookie) a podla toho mam cas
- nikde nic nepisem, nikde nic neodhadujem, proste ciste data z GPS alebo z hodin
- odberne miesta nemusia byt pomenovane, staci to spriemerovat na 100m stvorce a vyniest do mapy cluster s priemernym casom cakania
- uplne idealne este spravit aj viewer na pocitac kde si len pozries okolie a zistis kam vlastne chces podla aktualnych dat ist

**Comment by thebrownfox** [comment]
@4ndrej hej taky usecase dava zmysel. Vidim ale problem s casom, za aky to ma byt v produkcii.

**Comment by 4ndrej** [comment]
tusim bude aj druhy testing o tyzden

**Comment by stanop** [comment]
Nachystal som skript ktory vie rozsirit textovy zoznam testovacich miest o GPS suradnice staci mat python a API key na Google Geocoding API, v pripade ze by mal niekto zaujem dodam jednorazovo aj data.
Kod je tu https://gist.github.com/stanop/d1e9bd6593b141d94c8a566f2040da78
